cmd/pulumi-test-language/providers/simple_invoke_provider.go

```diff
@@ -232,6 +250,36 @@ func (p *SimpleInvokeProvider) Invoke(
 				"result": resource.NewStringProperty(value.StringValue() + " world"),
 			},
 		}, nil
+	} else if req.Tok == "simple-invoke:index:myInvokeScalar" {
+		value, ok := req.Args["value"]
+		if !ok {
+			return plugin.InvokeResponse{
+				Failures: makeCheckFailure("value", "missing value"),
+			}, nil
+		}
+
+		if value.IsComputed() {
+			return plugin.InvokeResponse{
+				// providers should not get computed values (during preview)
+				// since we bail out early in the core SDKs or generated provider SDKs
+				// when we encounter unknowns
+				Failures: makeCheckFailure("value", "value is unknown when calling myInvoke"),
```

Comment:
```suggestion
				Failures: makeCheckFailure("value", "value is unknown when calling myInvokeScalar"),
```
be careful with the copy-pasta

Reply:
yikes 😨